src/app/settings.py

```
@@ -108,11 +108,19 @@
 METADATA_URI = os.getenv("ECS_CONTAINER_METADATA_URI", None)
 IN_ECS = METADATA_URI is not None
 
+ECS_TASK_ARN = ""
+ECS_CONTAINER_ID = ""
 if IN_ECS:
     container_metadata = requests.get(METADATA_URI).json()
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)